### ISSUE TITLE
Add runtime support for Ethereum network configuration

### ISSUE THREAD

**Issue by kdeme** [status-im/nimbus-eth1#2387]
Support for Portal testnets got added here: https://github.com/status-im/nimbus-eth1/issues/2206

However, once different data than the mainnet data gets used on a Portal network, the node will also need to be able to select which network config to use per network.

**Comment by web3-developer** [comment]
I guess this config selection should be based on the selected `--network` so selecting a network that contains non-mainnet data would trigger reading in the matching config. 

One thing to think about is how should portal handle local networks when selecting `--network=none`. Currently we default to using mainnet config but what if you want to test different data in a private network.

**Comment by kdeme** [comment]
> One thing to think about is how should Fluffy handle local networks when selecting `--network=none`. Currently we default to using mainnet config but what if you want to test different data in a private network.

Yes, we don't have thought this through currently. In the case of network none we need extra way to say which config.

Perhaps we add an additional `--network-config` flag specifically for this setup.
Or additional info can get added in the network flag, e.g. `--network=none:mainnet` or something like that. Open to ideas here...
